android_client/app/src/main/AndroidManifest.xml

```diff
@@ -21,7 +21,7 @@
         <activity
             android:name=".MainActivity"
             android:configChanges="screenLayout|orientation|screenSize"
-            android:label="@string/app_name">
+            android:exported="true">
```

Comment:
Why don't we keep the `label` attr?

Reply:
Duplicate. We have declare that in application section.